lib/matplotlib/transforms.py

```diff
@@ -808,13 +808,17 @@ def from_bounds(x0, y0, width, height):
         return Bbox.from_extents(x0, y0, x0 + width, y0 + height)
 
     @staticmethod
-    def from_extents(*args):
+    def from_extents(*args, minpos=None):
         """
         Create a new Bbox from *left*, *bottom*, *right* and *top*.
 
-        The *y*-axis increases upwards.
+        The *y*-axis increases upwards. Optionally, passing *minpos* will set
+        that property on the returned Bbox.
```

Comment:
I think we should start documenting `minpos` where it is being used.  I think the suggestion below is correct, but feel free to make more precise

```suggestion
        The *y*-axis increases upwards. 
        
        *minpos* : float or None
           If this is passed the bbox can have a minimum value allowed for the bounds that will override
           user-supplied bounds.  This is useful when dealing with logarithmic scales and other scales where 
           negative bounds result in floating point errors.  
```